java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -101,7 +101,7 @@ public DType getDataType() {
     }
 
     @Override
-    public long getNumRows() {
+    public long getRowCount() {
```

Comment:
You cannot delete a public API like this.  We need to deprecate the old one.

Reply:
Sure, let me do that.

Reply:
done.